src/app/resources/resources.component.html

```diff
@@ -1,6 +1,11 @@
 <!-- FOR DEMO ONLY -->
 <!-- use the async pipe when fetching for database -->
-<button mat-raised-button routerLink="add" color="primary" routerLinkActive="active" i18n>Add Resource</button>
+<div *ngIf="nationname === ''; else nationName" class="list-item-action">
+  <button mat-raised-button routerLink="add" color="primary" routerLinkActive="active" i18n>Add Resource</button>
+</div>
+<ng-template #nationName>
+  <h1 i18n>{{nationname}} Resources</h1>
```

Comment:
can we have it like 'Resources : NationName' or for same planet it can be 'Resources: Local'